lablib/processors/ayon_ociolook_file.py

```diff
@@ -77,17 +88,65 @@ def load(self) -> None:
         for item in ops_data["data"]["ocioLookItems"]:
             self._sanitize_file_path(item, all_relative_files)
 
-        self.operator = AYONOCIOLookProduct.from_node_data(ops_data["data"])
+        self._process_look_file_to_color_operators(ops_data["data"])
+
+    def _process_look_file_to_color_operators(self, data: dict) -> None:
+        """Process the OCIO Look file to color operators.
+
+        Args:
+            data (dict): The OCIO Look data.
+        """
+
+        look_working_colorspace = data["ocioLookWorkingSpace"]["colorspace"]
+        look_items = data["ocioLookItems"]
+
+        for index, item in enumerate(look_items):
+            filepath = item["file"]
+            lut_in_colorspace = item["input_colorspace"]["colorspace"]
+            lut_out_colorspace = item["output_colorspace"]["colorspace"]
+            direction = item["direction"]
+            interpolation = item["interpolation"]
+
+            if index == 0:
+                # set the first colorspace as the current working colorspace
+                current_working_colorspace = look_working_colorspace
+
+            if current_working_colorspace != lut_in_colorspace:
+                self._operators.append(
+                    OCIO.ColorSpaceTransform(
+                        src=current_working_colorspace,
+                        dst=lut_in_colorspace,
+                    )
+                )
+
+            self._operators.append(
+                OCIO.FileTransform(
+                    src=Path(filepath).as_posix(),
+                    interpolation=operators.get_interpolation(interpolation),
+                    direction=operators.get_direction(direction),
+                )
+            )
+
+            current_working_colorspace = lut_out_colorspace
+
+        # making sure we are back in the working colorspace
+        if current_working_colorspace != look_working_colorspace:
+            self._operators.append(
+                OCIO.ColorSpaceTransform(
+                    src=current_working_colorspace,
+                    dst=look_working_colorspace,
+                )
+            )
 
     def get_oiiotool_cmd(self) -> List[str]:
         """Get arguments for the OIIO command."""
         args = []
-        for xfm in self.operator.to_ocio_obj():
-            if isinstance(xfm, OCIO.FileTransform):
-                lut = Path(xfm.getSrc()).resolve()
+        for op in self.color_operators:
```

Comment:
imho we should be careful about terminology.
as i understand we refer `self.color_operators` to OCIO objects and similar throughout the module. it might be easy to assume they'd be LabLib operators.

Reply:
Definitely! Perhaps, `ocio_objects` could be much better, right?

Reply:
yes exactly. i think it'd help readability

Reply:
Well it turns out that just the rename had produced quite a big list of changes. 

Anyway, during the changes I had finally relized what did you mean about unification of color and reposition methods by adding `to_oiio_args` method also to color operators. Yes that would be simply lovely to have it available in color. I will add it into issues.

Reply:
here it is https://github.com/ynput/LabLib/issues/40